test/commutativetests.jl

```diff
@@ -10,6 +10,7 @@ include("subs.jl")
 include("hash.jl")
 include("div.jl")
 include("norm.jl")
+include("var.jl")
```

Comment:
Just comment this out to make the test pass. If you put test_broken then it will break in TypedPolynomials and DynamicPolynomials once we make the tests pass :-P